setup.py

```diff
@@ -6,7 +6,7 @@
 
 setuptools.setup(
     name="sanpy",
-    version="0.0.20",
+    version="0.0.21",
```

Comment:
What is the logic behind the version system, do we use in sanpy?
Usually, the third value is the bugs/fixes.

Reply:
@Partysun For now we're just bumping the version so we can upload to pypi. But this is a good suggestion - we may start utilizing what you suggested: `major.minor.bugfixes`. So this PR should bump the minor. Also adding a CHANGELOG file

Reply:
https://semver.org